tensorflow-cpu.yaml

```diff
@@ -0,0 +1,122 @@
+#nolint:valid-pipeline-git-checkout-tag
+package:
+  name: tensorflow-cpu
+  version: 0_git20250106
+  epoch: 0
+  description: tensorflow-cpu
+  copyright:
+    - license: Apache-2.0
+  target-architecture:
+    - x86_64
+  dependencies:
+    runtime:
+      - bash
+      - build-base
+      - curl
+      - pkgconf
+      - py3.11-pip
+      - python-3.11
+      - unzip
+
+vars:
+  pypi-package: tensorflow-cpu
+  prefix: /usr
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - py3.11-pip
+      - python-3.11
+  environment:
+    # This is needed to work around the error "ValueError: ZIP does not support timestamps before 1980"
+    SOURCE_DATE_EPOCH: 315532800
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: d311fe017c036143186f9185f491fa6d6461a84e
+      repository: https://github.com/tensorflow/build
+      branch: master
+
+  - working-directory: tensorflow_runtime_dockerfiles
+    pipeline:
+      - name: Install cpu requirements
+        runs: |
+          python3 -m pip install \
+            --no-cache-dir \
+            --root ${{targets.contextdir}} \
+            --prefix ${{vars.prefix}} \
+            -r cpu.requirements.txt
+      - name: Install Tensorflow
+        runs: |
+          python3 -m pip install \
+            --no-cache-dir \
+            --root ${{targets.contextdir}} \
+            --prefix ${{vars.prefix}} \
+            ${{vars.pypi-package}}
+      - name: Install jupyter requirements
+        runs: |
+          python3 -m pip install \
+            --no-cache-dir \
+            --root ${{targets.contextdir}} \
+            --prefix ${{vars.prefix}} \
+            -r jupyter.requirements.txt \
+            -U
+      - name: Install bashrc
+        runs: |
+          mkdir -p ${{targets.contextdir}}/etc
+          cp bashrc ${{targets.contextdir}}/etc/bash.bashrc
+      - name: Install tensorflow tutorials
+        runs: |
+          mkdir -p ${{targets.contextdir}}/tf/tensorflow-tutorials
+          cp jupyter.readme.md ${{targets.contextdir}}/tf/tensorflow-tutorials/README.md
+      - name: Cleanup pycache
+        runs: |
+          find . -name '__pycache__' -exec rm -rf {} +
+
+test:
+  environment:
+    contents:
+      packages:
+        - curl
```

Comment:
Do we need this here if we have it as a runtime dep 🤔 

P.S. Please correct me if I got that wrong!

Reply:
It's needed for the tests for Jupyter